src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -1934,31 +1937,46 @@ def create_vnet_gateway(resource_group_name, virtual_network_gateway_name, publi
     if asn or bgp_peering_address or peer_weight:
         vnet_gateway.enable_bgp = True
         vnet_gateway.bgp_settings = BgpSettings(asn, bgp_peering_address, peer_weight)
-    if address_prefixes:
+
+    if any((address_prefixes, radius_secret, radius_server, client_protocol)):
```

Comment:
redundant extra brackets here?

Reply:
No, the outer brackets are the method brackets. The inner brackets define the list generator. I figured this should be more performant than `any([a, b, c])` because Python should be able to stop iterating when it finds the first True instead of creating a whole list object and then iterating through that.  But I could be totally wrong.